pkg/api/v1/defaults_test.go

```diff
@@ -575,11 +575,11 @@ func TestSetDefaultNodeStatusAllocatable(t *testing.T) {
 		if rl == nil {
 			return nil
 		}
-		copy := make(v1.ResourceList, len(rl))
+		copy_ := make(v1.ResourceList, len(rl))
```

Comment:
I would like to avoid changing things for that reason, unless the project has adopted a specific style rule. It's difficult to keep the code consistent othewise.